components/clubs-page.jsx

```diff
@@ -131,6 +130,10 @@ var ClubsPage = React.createClass({
         </HeroUnit>
         <section>
           <WebLitMap/>
+          <div className="mapDiv" id="mapDivID">
+            <Map accessToken={process.env.MAPBOX_ACCESS_TOKEN || 'pk.eyJ1IjoiYWxpY29kaW5nIiwiYSI6Il90WlNFdE0ifQ.QGGdXGA_2QH-6ujyZE2oSg'}
+           mapId={process.env.MAPBOX_MAP_ID || 'alicoding.ldmhe4f3'} mapParentID={'mapDivID'} mapClassName={'mapDivChild'}/>
```

Comment:
We should probably get rid of `mapParentID` and the `id` on line 133 since you're not using them anymore?